docs/source/contents/4_draft_mapping.rst

```diff
@@ -1194,7 +1194,7 @@ The majority of the applicable values are associate with a `@valueURI`.  The `re
 
 .. code-block:: turtle
 
-    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
```

Comment:
Thanks for these additions - I think they are double accounted for in my PR as well.